src/foam/lib/parse/InvalidPStream.java

```diff
@@ -38,11 +43,21 @@ public PStream tail() {
   public Object value() {
     return null;
   }
+    
+  @Override
+  public Object operator() {
```

Comment:
"operator" is very specific to query parsing, but PStream is supposed to be used for all parsers.  We shouldn't add "operator" support here, but rather just implement the behaviour we need in the appropriate query parser.